doc/rtd/reference/datasources/nocloud.rst

```diff
@@ -138,26 +273,41 @@ Where ``scheme`` can be ``ftp`` or ``ftps``, ``userinfo`` will be
 ``host`` can be an IP address or DNS name, and ``port`` is which network
 port to use (default is ``21``).
 
-Seedfrom: Files
----------------
+Meta-configuration considerations
+=================================
+
+Above, we describe the two methods of providing meta-configuration (system
+configuration and line configuration). Two methods exist because there are
+advantages and disadvantages to each option, neither is clearly a better
+choice - so it is left to the user to decide.
+
+Line configuration
+------------------
 
-The path pointed to by the URI can contain the following
-files:
+Advantages
+^^^^^^^^^^
```

Comment:
I would probably not use sub-subheaders here - there aren't enough bullet points to make it a true section, so I'd probably turn the Advantages/Disadvantages in each case into regular text, but in bold.